msal/tests/Test.MSAL.NET.Unit/TestConstants.cs

```diff
@@ -42,14 +42,20 @@ class TestConstants
         public static readonly SortedSet<string> ScopeForAnotherResource = new SortedSet<string>(new[] { "r2/scope1", "r2/scope2" });
         public static readonly SortedSet<string> CacheMissScope = new SortedSet<string>(new[] { "r3/scope1", "r3/scope2" });
         public static readonly string ScopeForAnotherResourceStr = "r2/scope1 r2/scope2";
-        public static readonly string ProductionPrefNetworkEnvironment = "login.microsoftonline.com";
-        public static readonly string ProductionPrefCacheEnvironment = "login.windows.net";
+        public static readonly string Uid = "my-uid";
+        public static readonly string Utid = "my-utid";
+
+        public static readonly string ProdPrefNetworkEnv = "login.microsoftonline.com";
```

Comment:
nit: not sure I understand the need for shortening the consts.

Reply:
ok, will change back

Reply:
Seems like these are still not changed.

---
In reply to: [223459786](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1304#discussion_r223459786) [](ancestors = 223459786)